DigitalLearningSolutions.Web/Helpers/TrackingSystem/CentrePage.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers.TrackingSystem

```

Comment:
Given that the page enums are relevant to the Tracking System subfolders, can we put them into the relevant folders?